core/src/test/scala/org/apache/spark/ThreadingSuite.scala

```diff
@@ -130,7 +131,7 @@ class ThreadingSuite extends SparkFunSuite with LocalSparkContext {
               Thread.sleep(100)
             }
             if (running.get() != 4) {
-              println("Waited 1 second without seeing runningThreads = 4 (it was " +
+              logInfo("Waited 1 second without seeing runningThreads = 4 (it was " +
```

Comment:
Oh darn, maybe the logging ends up dragging in some reference to the outer class. Maybe just zap this whole clause, or make the error message a field in this `ThreadingSuiteState` and use it below.